awebox/mdl/aero/indicators.py

```diff
@@ -80,138 +87,149 @@ def get_performance_outputs(options, atmos, wind, variables, outputs, parameters
+    outputs['aerodynamics']['wingtip_int' + str(kite)] = q - ehat_span * b_ref / 2.
 
-        if parent > 0:
-            grandparent = architecture.parent_map[parent]
-            q_parent = variables['xd']['q' + str(parent) + str(grandparent)]
-        else:
-            q_parent = cas.DM.zeros((3, 1))
+    outputs['aerodynamics']['fstar_aero' + str(kite)] = cas.mtimes(air_velocity.T, ehat_chord) / c_ref
 
-        tether = q - q_parent
-        radius_vec = tether - cas.mtimes(tether.T, n_hat) * n_hat
-        radius = vect_op.norm(radius_vec)
-        varrho = radius / b_ref
+    outputs['aerodynamics']['r' + str(kite)] = kite_dcm.reshape((9, 1))
 
-        lift_betz_scale = 4. * np.pi * rho * b_ref * radius**3. / (radius + 0.5 * b_ref) * u_infty**3. / tangential_speed
+    if int(options['kite_dof']) == 6:
+        outputs['aerodynamics']['m_aero_body' + str(kite)] = m_aero_body
 
-        mu_center = varrho / (varrho + 0.5)
-        tip_speed_ratio = kite_speed_ratio / mu_center
+    outputs['aerodynamics']['mach' + str(kite)] = get_mach(options, atmos, air_velocity, q)
+    outputs['aerodynamics']['reynolds' + str(kite)] = get_reynolds(options, atmos, air_velocity, q, parameters)
 
-        a_betz = 1./3.
+    return outputs
 
-        mu_min = (varrho - 0.5)/(varrho + 0.5)
-        n_steps = 20
-        delta_mu = (1. - mu_min) / n_steps
 
-        factor = 0.
-        for mdx in range(n_steps - 1):
-            mu = mu_min + (delta_mu / 2.) + np.float(mdx) * delta_mu
+def get_vortex_verification_f_lift_earth(dq_kite):
+    f_lift_earth = cas.DM(4.86123e6) * vect_op.xhat()
+    ehat_theta = vect_op.normalize(dq_kite)
+    u_downstream = 6.6666;
+    u_tangential = 47.4918;
+    air_velocity = u_downstream * vect_op.xhat() + u_tangential * ehat_theta;
+    return f_lift_earth, air_velocity
 
-            a_prime_betz = a_betz * (1. - a_betz) / tip_speed_ratio ** 2. / mu**2.
-            new_factor = np.sqrt((1. - a_betz)**2. + (tip_speed_ratio * mu * ( 1 + a_prime_betz))**2. )
-            factor += new_factor * delta_mu
 
-        # lift_betz_optimal = factor * lift_betz_scale
-        lift_betz_optimal = cas.DM(5.0129 * 1.e6)
-        outputs['aerodynamics']['f_lift_verification' + str(kite)] = lift_betz_optimal
 
-        ehat_span = outputs['aerodynamics']['ehat_span' + str(kite)]
-        gamma_verification = lift_betz_optimal / b_ref / rho / vect_op.norm(vect_op.cross(ua, ehat_span))
-        outputs['aerodynamics']['gamma_verification' + str(kite)] = gamma_verification
-        outputs['aerodynamics']['gamma' + str(kite)] = gamma_verification
+def collect_vortex_verification_outputs(options, architecture, atmos, wind, variables, parameters, intermediates, outputs):
 
     return outputs
 
-def collect_power_balance_outputs(variables, n, outputs, architecture):
+def collect_power_balance_outputs(options, architecture, variables, intermediates, outputs):
```

Comment:
sorry, i'm not sure what this ditto means. 
are you asking me to re-name this? because power_balance is what this thing is called in the quality checks...

Reply:
It means that I find the variable name `intermediates` to be fuzzy and very non-descriptive

Reply:
ok! thanks. then, fixed!